org-custom-cookies.el

```diff
@@ -162,6 +162,19 @@ top-level heading."
   (cl-loop for (regex . callback) in org-custom-cookies-alist
            do (org-custom-cookies--update-current-heading-cookie regex callback)))
 
+(defun org-custom-cookies--update-cookie-ctrl-c-ctrl-c ()
+"Update the custom cookie under the cursor using `org-ctrl-c-ctrl-c'.
+This will update any org custom cookie in the fashion as with default
+org cookies.
+
+Hook this function to `org-ctrl-c-ctrl-c-hook' for it to work."
+  (catch 'updated-cookie
+    (cl-loop for (regex . callback) in org-custom-cookies-alist
+             do (if (org-in-regexp regex)
+                    (progn
+                      (org-custom-cookies--update-nearest-heading-cookie regex callback)
+                      (throw 'updated-cookie 1))))))
```

Comment:
I'm not an elisp expert, but I think this function might be simpler if you remove the `catch` and replace the `throw` with `cl-return`? And maybe return `t` instead of 1?

If not, let me know, and I'll merge it as is.

Reply:
Done in 1186aca !